### PR TITLE
HTML STRUCTURE version 1.0.0

### PR DESCRIPTION
**Pull request by KingLyrics**
Hey @gmagnenat , this is  the Drafted HTML STRUCTURE VERSION 1.0.0. Let's see if we can make any corrections to the file before moving forward.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,27 +1,121 @@
...
+	<link href="https://fonts.googleapis.com/css2?family=IBM+Plex+Sans:wght@400;700&display=swap" rel="stylesheet" />
+	<link rel="stylesheet" href="css/main.css" />
+	<title>Frontend Mentor | Equalizer landing page</title>
+</head>
+
+<body>
+
+	<!-- Page container if needed -->
+
+	<div class="container">
```
Comment (gmagnenat) [inline public/index.html:19]: Not sure we need it. A container inside the sections would make sense.

File: public/index.html [hunk 2]
```diff
@@ -1,27 +1,121 @@
...
+
+		<!-- Navbar to store the logo and the art at the very end of the page -->
+		<nav class="navbar">
```
Comment (gmagnenat) [inline public/index.html:22]: Maybe wrap that in a header instead as there aren't really a navigation here.

File: public/index.html [hunk 3]
```diff
@@ -1,27 +1,121 @@
...
+			<div class="logo">
+				<img src="./assets/logo.svg" alt="">
+			</div>
+			<!-- Navbar should have a position of relative so that the artwork can be positioned absolute or can be flexed -->
+			<div class="artwork">
```
Comment (gmagnenat) [inline public/index.html:27]: I don't think we need this. It's decorative and can be added as background

File: public/index.html [hunk 4]
```diff
@@ -1,27 +1,121 @@
...
+				<img src="./assets/bg-pattern-1.svg" alt="">
+			</div>
+		</nav>
+		<!-- End of Navbar -->
+
+		<!-- Section that contains all the texts  -->
+		<section class="hero-texts">
+			<div class="lead-text">
```
Comment (gmagnenat) [inline public/index.html:35]: The div doesn't seem necessary. I think we can work with h1 and p directly

File: public/index.html [hunk 5]
```diff
@@ -1,27 +1,121 @@
...
+				<h1>We make your music sound extraordinary</h1>
+			</div>
+			<div class="lead-body">
```
Comment (gmagnenat) [inline public/index.html:38]: Same as above

File: public/index.html [hunk 6]
```diff
@@ -1,27 +1,121 @@
...
+					sounds with a professional grade parametric EQ & volume mixer. Control bass,
+					mids, treble, gain control, reverb, and more!</p>
+			</div>
+		</section>
+		<!-- End of section texts -->
+
+		<!-- Section of the hero images -->
+		<!-- The black background has me questioning if the blue art inside should be in it or hero-imgs should just be set to absolute. -->
+		<section class="hero-imgs">
+			<div class="black-bg">
```
Comment (gmagnenat) [inline public/index.html:50]: Not necessary. We can use a background color on the section.

File: public/index.html [hunk 7]
```diff
@@ -1,27 +1,121 @@
...
+			<div class="blue-art">
+				<img src="./assets/bg-pattern-2.svg" alt="">
+			</div>
+
+			<div class="app-img">
+				<img src="./assets/illustration-app.png" alt="">
+			</div>
+			<!-- Orange card begins -->
+			<div class="card">
+				<article class="card-title">
```
Comment (gmagnenat) [inline public/index.html:61]: Article not necessary. Just an h2 with the class card-title

File: public/index.html [hunk 8]
```diff
@@ -1,27 +1,121 @@
...
+				</article>
+				<article class="card-body">
+					<p>
+						Get expert-level
+						control with a robust equalizer, volume mixer, and spatial audio. Take your
+						listening experience to a whole new level and access all our incredible
+						features!
+					</p>
+
+					<article class="pricing">
```
Comment (gmagnenat) [inline public/index.html:72]: I have to read about that, but not sure article is the proper tag for this composition..

File: public/index.html [hunk 9]
```diff
@@ -1,27 +1,121 @@
...
+						<div class="money">
+							<h1>$4</h1>
+						</div>
```
Comment (gmagnenat) [inline public/index.html:75]: Can be simplified with spans

File: public/index.html [hunk 10]
```diff
@@ -1,27 +1,121 @@
...
+						<div class="month">
+							<h5>/month</h5>
+						</div>
+					</article>
+
+					<div class="btns">
+						<div class="btn-ios">
+							<h1><img src="./assets/ihts-apple.svg" alt=""> ios Download
+						</div>
```
Comment (gmagnenat) [inline public/index.html:84]: Simply use buttons here. We can place the icon as a pseudo element

File: public/index.html [hunk 11]
```diff
@@ -1,27 +1,121 @@
...
+				<p>
+					All rights reserved &copy;Equalizer 2022
+					Have any problems? Contact us via social media or email us at
+					<span class="bold">
+						equalizer@example.com
+					</span>
+				</p>
+			</div>
+			<div class="socials">
+				<img src="./assets/icon-facebook.svg" alt="">
```
Comment (gmagnenat) [inline public/index.html:111]: a links would be appropriate here

**Review by gmagnenat — COMMENTED** [review]
@KingLyrics I added a few comment by quickly browsing through the file. 

Everything is open to suggestions of course.

As a global comment, there are to many h1 in your structure and a few parts can be simplified by removing the wrapping divs.

**Comment by KingLyrics** [comment]
@gmagnenat I will get on the changes immediately and also let you know if I think anything also should be changed.

File: public/index.html (hunk 2 above)
Comment (KingLyrics) [reply public/index.html:22]: That's true. Just made the correction now

File: public/index.html (hunk 3 above)
Comment (KingLyrics) [reply public/index.html:27]: We don't need the logo or the artwork?

File: public/index.html (hunk 4 above)
Comment (KingLyrics) [reply public/index.html:35]: Alright. The code has been corrected.

File: public/index.html (hunk 5 above)
Comment (KingLyrics) [reply public/index.html:38]: Corrected as well

**Comment by gmagnenat** [comment]
@KingLyrics great 👍. Please don't merge the pull request before the review. To keep the process clean ;)

File: public/index.html (hunk 1 above)
Comment (KingLyrics) [reply public/index.html:19]: Alright noted and changes have been made

File: public/index.html (hunk 4 above)
Comment (KingLyrics) [reply public/index.html:35]: `<section class="hero-texts">
		<div class="container">
			<h1> We make your music sound extraordinary</h1>
			<p>
				A system audio equalizer
				specifically designed for Android and iOS. Freely tune the way your music
				sounds with a professional grade parametric EQ & volume mixer. Control bass,
				mids, treble, gain control, reverb, and more!
			</p>
		</div>
	</section>
`

**Comment by KingLyrics** [comment]
@gmagnenat Really sorry about that. I noticed it a bit too late. Thank you.

File: public/index.html (hunk 6 above)
Comment (KingLyrics) [reply public/index.html:50]: Alright changes have been made.

File: public/index.html (hunk 7 above)
Comment (KingLyrics) [reply public/index.html:61]: Alright resolved.

File: public/index.html (hunk 8 above)
Comment (KingLyrics) [reply public/index.html:72]: Well, I changed  them to div. I don't know if that will be better?

File: public/index.html (hunk 9 above)
Comment (KingLyrics) [reply public/index.html:75]: Corrected

File: public/index.html (hunk 10 above)
Comment (KingLyrics) [reply public/index.html:84]: Corrected.

File: public/index.html (hunk 11 above)
Comment (KingLyrics) [reply public/index.html:111]: Corrected.

**Comment by gmagnenat** [comment]
@KingLyrics something I should have mentionned before... 
The css and html is done in the src folder. Everthing is copied and compiled in the public directory with the build script.
Locally just write ```yarn install``` at the root of the project to install all node modules. Then run ```yarn start``` to start the script, local server and live reload.

File: public/index.html (hunk 3 above)
Comment (KingLyrics) [reply public/index.html:27]: Corrected

**Comment by KingLyrics** [comment]
@gmagnenat Oh no. So all the work I did was in the public folder? Do I just copy and paste the code into the src index?

**Comment by KingLyrics** [comment]
@gmagnenat just ran `yarn start` worked and live reload is up. Files have been built into the public folder

**Comment by gmagnenat** [comment]
@KingLyrics no harm :) yes, just copy the html in there.

**Pull request merged**